modules/wri_common/wri_common.module

```diff
@@ -188,6 +196,28 @@ function wri_common_tokens($type, $tokens, array $data, array $options, Bubbleab
       }
     }
   }
+  if ($type == 'node') {
+    if (isset($data['node']) && $node = $data['node']) {
+      foreach ($tokens as $name => $original) {
+        // Find the desired token by name.
+        switch ($name) {
```

Comment:
Why a switch here if there's only one case? Is it to account for similar future functionality for additional not yet known cases?

Reply:
It's a good question -- It seems to be a convention with tokens to always use a switch statement, but that's probably just because a lot of people have copied the default example: https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/lib/Drupal/Core/Utility/token.api.php#L74

In practice, I've often had to add another case later in the game, so it's nice to have a switch statement all set up and ready to take more cases. But an if/then statement would work fine too.